src/packages/admin-ui/src/pages/list/graphql.ts

```diff
@@ -8,7 +8,7 @@ export const queryForEntityPage = (entityName: string, entityByType: (type: stri
 	const queryName = pluralName[0].toLowerCase() + pluralName.slice(1);
 
 	return gql`
-		query AdminUIListPage($filter: ${pluralName}ListFilter, $pagination: ${pluralName}PaginationInput) {
+		query AdminUI${entityName}ListPage($filter: ${pluralName}ListFilter, $pagination: ${pluralName}PaginationInput) {
```

Comment:
@steve-keep you should look at this pattern before we introduce it.

When creating a new entity we want to update the table so that the new entity is shown. We've opted to refetched the query that populates the table. This query was generically named, so it was refetching all the entities in all the tables. Now we've made it specific to the entity so we only fetch the table that's changed.

@siowhoi  did I summarize this correctly?

Reply:
Yes, that's correct.